packages/opentelemetry-core/src/platform/node/timer-util.ts

```diff
@@ -13,6 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-export function unrefTimer(timer: NodeJS.Timeout): void {
+export function unrefTimer(timer: NodeJS.Timer): void {
```

Comment:
@mayurkale22 Are you aware of anything that would cause this to be an issue? This is the only change needed for this PR to go green.

Reply:
The Timer type exists specifically to have compatibility with older versions of node. This should be ok.